Source/Scene/Model.js

```diff
@@ -156,8 +156,49 @@ define([
         this.skinnedNodesIds = [];
     }
 
+    LoadResources.prototype.getBufferBytes = function(buffer, byteOffset, byteLength) {
+        return getSubarray(buffer, byteOffset, byteLength);
+    };
+
     LoadResources.prototype.getBuffer = function(bufferView) {
-        return getSubarray(this.buffers[bufferView.buffer], bufferView.byteOffset, bufferView.byteLength);
+        return this.getBufferBytes(this.buffers[bufferView.buffer], bufferView.byteOffset, bufferView.byteLength);
+    };
+
+    LoadResources.prototype.getDecodedBuffer = function(bufferView, decodeBufferAccessors) {
+        var bufferData = new Uint8Array(0);
+        var byteShift = 0;
+        for (var i in decodeBufferAccessors) {
```

Comment:
With `for...in`, we always use `if (decodeBufferAccessors.hasOwnProperty(i)) {`.  Maybe it's OK without in this case, but I would rather be consistent.